src/main/java/application/rest/v1/actions/FunctionResolver.java

```diff
@@ -69,13 +69,14 @@ public String resolve(ResolutionContext context, String suffix) {
                 }
                 // One of the function parameters couldn't be resolved.
                 else {
+                    // if throw exception here breaking UI action menu view kibana log link
                     return null;
```

Comment:
Breaks how?  This is temporary, right?  We don't want comments like this forever in the code.   I think we need to review this.